MahApps.Metro/Themes/Tile.xaml

```diff
@@ -46,7 +47,10 @@
                                         FontSize="{Binding TitleFontSize, RelativeSource={RelativeSource TemplatedParent}}"
                                         Margin="3" />
                         </Label>
-                    </Grid>
+                    </Grid>  
```

Comment:
Please pay attention for a **correct indent.**
Indents as you know are really important for the coding style!